src/app/pages/PerpetualPage/slice.ts

```diff
@@ -51,6 +52,9 @@ const perpetualPageSlice = createSlice({
     reset(state) {
       state = initialState;
     },
+    setToastedTransactions(state, { payload }: PayloadAction<string>) {
```

Comment:
set* implies changing the whole variable value, use push instead.
```suggestion
    pushToastedTransaction(state, { payload }: PayloadAction<string>) {
```